contracts/cw1155-base/src/lib.rs

```diff
@@ -9,6 +9,9 @@ pub use crate::state::Cw1155Contract;
 use cosmwasm_std::Empty;
 use cw1155::Cw1155QueryMsg;
 
+// todo - is this standard?
+pub const CW_ADDRESS_LENGTH: usize = 62;
```

Comment:
todo - check this

Reply:
nvm, removing TokenKey since it's unnecessary - using token id as primary key instead